docs/user-guide/testing-your-assistant.rst

```diff
@@ -15,7 +15,7 @@ Testing Your Assistant
    If you are looking to tune the hyperparameters of your NLU model,
    check out this `tutorial <https://blog.rasa.com/rasa-nlu-in-depth-part-3-hyperparameters/>`_.
 
-.. _end_to_end_testing:
+.. _end-to-end-testing:
```

Comment:
I checked the Rasa & rasa x docs and couldn't find any references that would break - they're all `<https://rasa.com/docs/rasa/user-guide/testing-your-assistant/#end-to-end-testing>`

Reply:
aha, sweet. if they're in the rasa docs it's probably best to change them to the `ref` instead though, as that's more reliable